lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -632,7 +632,8 @@ def write(chunk)
           end
           combined_entries.concat(request[:entries])
         end
-        @write_request.call(entries: combined_entries)
+        @write_request.call(entries: combined_entries) unless
```

Comment:
If all entries in this chunk is invalid and dropped, we should not make a request. This fix is to make the behavior consistent with how we deal with it when `split_logs_by_tag` is true.

Our unit test `test_malformed_log` actually caught this.